source/codegen/common_helpers.py

```diff
@@ -33,7 +33,7 @@ def get_input_and_output_custom_types(functions):
   return (input_custom_types, output_custom_types)
 
 def is_string_arg(parameter):
-  return parameter['type'] in ['ViChar[]', 'ViInt8[]', 'ViUInt8[]']
+  return parameter['type'] in ['ViChar[]', 'ViInt8[]', 'ViUInt8[]'] or parameter['grpc_type'] == 'string'
```

Comment:
I'm guessing this will go away from the diff when you take in Greg's changes so it becomes `return parameter['grpc_type'] in ['string', 'bytes']`, right?